src/Writer/AbstractFeed.php

```diff
@@ -176,11 +182,14 @@ public function setDateModified($date = null)
     /**
      * Set the feed last-build date. Ignored for Atom 1.0.
      *
-     * @param  null|int|DateTimeInterface
-     * @return $this
+     * @param null|int|DateTimeInterface
+     * @param DateTime|\DateTimeImmutable|int|null|string $date
+     *
+     * @return self
+     *
      * @throws Exception\InvalidArgumentException
      */
-    public function setLastBuildDate($date = null)
+    public function setLastBuildDate($date = null): self
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.